### ISSUE TITLE
Ziel(gruppe) des Tools

### ISSUE THREAD

**Issue by Hartz-Arbeit** [Hartz-Arbeit/LohntDat#7]
Ich hatte mir das ganze als kleine App mit Sprengkraft vorgestellt, scheinbar geht die Vorstellung hier jedoch größtenteils zu einer App für die tatsächlich genaue Berechnung (mit Immos, Aktien). Deshalb eine kurze Umfrage zur Klärung.

https://strawpoll.de/e41ds2f

**Comment by jpv-os** [comment]
Ich stimme zu, kleine App mit viel Message dahinter. Auf der technischen Seite bereiten wir uns bereits auf die Berechnung vor, die in jedem Fall im Hintergrund geschehen muss. Die Kunst wäre es nun, die komplexe Eingabe auf das wesentliche zu reduzieren, sodass ein Nutzer das Formular in 1-3 Minuten ausfüllen kann.
